extensions/mongodb-client/runtime/src/test/java/io/quarkus/mongodb/DeploymentWatchTest.java

```diff
@@ -17,14 +17,15 @@
 
     //    @Test
     void test() throws InterruptedException {
-        MongoClient mongoClient = MongoClients.create("mongodb://localhost:27018");
-        MongoDatabase database = mongoClient.getDatabase("test");
-        MongoCollection<Document> collection = database.getCollection("restaurants");
-        new Thread(() -> collection.watch().forEach(printBlock)).start();
-        Document doc = new Document().append("hello", "world");
-        collection.insertOne(doc);
-        collection.updateOne(eq("hello", "world"), new Document("$set", new Document().append("hello", "foo")));
-        new Thread(() -> collection.watch().forEach(printBlock)).start();
-        Thread.sleep(5000);
+        try (MongoClient mongoClient = MongoClients.create("mongodb://localhost:27018")) {
```

Comment:
I can't remember when I wrote this. I believe this should be dropped. It's not even running in the test suite.